README.md

```diff
@@ -93,9 +93,18 @@ from anywhere.
 
 ### Requirements
 
-Before running any kind of analysis, you need to install some analysis tools that HAROS
+Before running any kind of analysis, you need to install some analysis tools and plugins that HAROS
 uses behind the curtains. Install these *$dependencies$* with the following commands.
 
+For each *req* in [requirements.txt](requirements.txt):
+*(Not necessary if you install HAROS from `pip`)*
+
+```bash
+pip install req
```

Comment:
Would you not want to just run:

```
pip install -r requirements.txt
```

instead?

Reply:
Indeed. I will change that.